package.json

```
@@ -1,16 +1,15 @@
 {
   "name": "@hubmap/vitessce-image-viewer",
-  "version": "0.0.7",
-  "main": "dist/index.js",
-  "module": "dist/index.js",
-  "jsnext:main": "dist/index.js",
+  "version": "0.0.8",
+  "main": "dist/index.umd.js",
+  "module": "dist/index.es5.js",
   "files": [
     "dist",
     "src"
   ],
   "description": "High bit depth, high resolution, many channels, low overhead",
   "scripts": {
-    "build-component": "mkdir dist || rm -r dist && node_modules/.bin/webpack --config webpack.component.prod.js --mode production",
+    "build-component": "mkdir dist || rm -r dist && rollup -c rollup.config.js",
```

```suggestion
    "build-component": "mkdir dist || rm -r dist && rollup --config",
```
- Preference for spelling out flags in scripts (unless it's obvious).
- If we're using the default, I don't think it's required? https://rollupjs.org/guide/en/